homeassistant/components/nut/const.py

```diff
@@ -0,0 +1,135 @@
+"""The nut component."""
+from homeassistant.const import POWER_WATT, TEMP_CELSIUS, TIME_SECONDS, UNIT_PERCENTAGE
+
+DOMAIN = "nut"
+
+PLATFORMS = ["sensor"]
+
+
+DEFAULT_NAME = "NUT UPS"
+DEFAULT_HOST = "localhost"
+DEFAULT_PORT = 3493
+
+KEY_STATUS = "ups.status"
+KEY_STATUS_DISPLAY = "ups.status.display"
+
+PYNUT_DATA = "data"
+PYNUT_STATUS = "status"
+PYNUT_UNIQUE_ID = "unique_id"
+PYNUT_MANUFACTURER = "manufacturer"
+PYNUT_MODEL = "model"
+PYNUT_FIRMWARE = "firmware"
+
+SENSOR_TYPES = {
+    "ups.status.display": ["Status", "", "mdi:information-outline"],
+    "ups.status": ["Status Data", "", "mdi:information-outline"],
+    "ups.alarm": ["Alarms", "", "mdi:alarm"],
+    "ups.time": ["Internal Time", "", "mdi:calendar-clock"],
+    "ups.date": ["Internal Date", "", "mdi:calendar"],
+    "ups.model": ["Model", "", "mdi:information-outline"],
+    "ups.mfr": ["Manufacturer", "", "mdi:information-outline"],
+    "ups.mfr.date": ["Manufacture Date", "", "mdi:calendar"],
+    "ups.serial": ["Serial Number", "", "mdi:information-outline"],
+    "ups.vendorid": ["Vendor ID", "", "mdi:information-outline"],
+    "ups.productid": ["Product ID", "", "mdi:information-outline"],
+    "ups.firmware": ["Firmware Version", "", "mdi:information-outline"],
+    "ups.firmware.aux": ["Firmware Version 2", "", "mdi:information-outline"],
```

Comment:
This doesn't seem like information that is important to expose ?

Reply:
Can drop them if we want to make this a breaking change.

Reply:
I saw we do. No one would care about this .

Reply:
nuked